bumpversion/__init__.py

```diff
@@ -152,7 +152,9 @@ def tag(cls, name, message):
         command = ["git", "tag", name]
         if message:
             command += ['--message', message]
-        subprocess.check_output(command)
+        elif message is None:
+            command += ['--annotate']
+        subprocess.call(command)
```

Comment:
This doesn't check the exit status like check_output did. There is check_call. Since the project is marked Python 2.7+ I don't think that'll be a problem.